.gitignore

```diff
@@ -95,3 +95,6 @@ bundler.d/*
 # yardoc
 /doc/
 .yardoc
+.DS*
+.DS_Store
```

Comment:
These two are redundant

Reply:
If there are other types of `DS_` files besides `DS_Store`, I've never encountered them.